filter_envoy/aggr_filter.rs

```diff
@@ -0,0 +1,125 @@
+use rpc_lib::rpc::Rpc;
```

Comment:
This is in the wrong folder. Also shouldn't this be generated?

Reply:
Yes  That's the default location, since everything is default generated to envoy, and I accidentally included a generated version.  Should be gone now.